doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -219,6 +219,59 @@ and ``coqtop``, unless stated otherwise:
 :-batch: Exit just after argument parsing. Available for ``coqtop`` only.
 :-verbose: Output the content of the input file as it is compiled.
   This option is available for ``coqc`` only.
+:-native-compiler (yes|no|ondemand): Enable the :tacn:`native_compute`
+  reduction machine and precompilation to ``.cmxs`` files for future uses
+  by :tacn:`native_compute`.
+  ``yes`` and ``no`` enable/disable both :tacn:`native_compute` (any call
+  to it will default back to :tacn:`vm_compute`) and precompilation.
```

Comment:
I would split this sentence into two parts, to make it a bit less confusing: "Setting `yes` enables `native_compute`; it also causes Coq to precompile the native code for future uses; all dependencies need to have been precompiled beforehand. Setting `no` disables `native_compute`; no files are precompiled.